### DISCUSSION TITLE
Server hacking

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#2439] [General]
It's really sad to see thats there is no anti cheat to prevent hackers spawning items or killing everyone with 10000000000 damage

**Comment by hakusaro** [comment]
TShock is a volunteer effort to fix the game. We do the best we can. You’ve been consistently negative and have done nothing to contribute to this community other than to bash us or complain about problems we can’t solve. Terraria is a game by ReLogic. We are not ReLogic. We cannot fix the game protocol that they designed. We can try to deal with some things, but we aren’t perfect.

You are no longer welcome here. We have warned you multiple times. We do not appreciate the your posts. We are volunteers working for free because we like the game and we like the people who use our software. We are not responsible for the game, nor are we obligated to take your harassment. This is just one of many posts, and you have already been banned from discord for similarly terrible behavior. We are not tolerating this any further.
